test/extended/openstack/servers.go

```diff
@@ -152,8 +118,46 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenStack pla
 	})
 })
 
-// listWorkerMachineSets lists all worker machineSets
-func listWorkerMachineSets(dc dynamic.Interface) ([]objx.Map, error) {
+// Checks that the Openstack created instances are as specified by the MachineSets
+func checkMachineSets(dc dynamic.Interface, computeClient *gophercloud.ServiceClient, machineSets []objx.Map) {
```

Comment:
On this function you are rather checking that the machinesets'  specs are satisfied on the existing openstack instances. 

As this change is interesting for covering CPMS validations in coming patches... I am wondering if you want to check exactly the same things you are checking on the workers for the masters. I believe you'll want to do different checks. 

For example, in CPMS you should have one single machineset, so passing a list of machinesets is not applicable in that case. In fact, you should check that only one machineset should be there. Also, in the CPMS you'll have the failureDomain sections, and in worker machinesets you should not.

Therefore, I think it's cleaner to create separate checks for the CP machineset than creating a function for both worker and master machinesets.

Reply:
Maybe renaming can be beneficial. I think that we can add another function just for masters and keep a function for the common code between masters and workers

Reply:
ok! in such a case, could you please then rename the function to something like commonCheckMachineSets and add a comment explaining that?

Reply:
yes